exonum/src/runtime/types.rs

```diff
@@ -46,6 +46,9 @@ pub type InstanceId = u32;
 /// Identifier of the method in the service interface required for the call.
 pub type MethodId = u32;
 
+/// Uniform presentation of a `Caller`.
+pub type CallerAddress = Hash;
```

Comment:
Yeah, probably. Although it doesn't on its own answer the discussion question whether we need some error-detecting encoding for addresses (I *think* it could be added later.)

Reply:
Agree, that we'll be able to change it in the future. If we will need it.